default.py

```diff
@@ -2,7 +2,9 @@
 ''' This is the actual InputStream Helper API script entry point '''
 
 from __future__ import absolute_import, division, unicode_literals
+import os
 import sys
-from api import run
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'lib'))  # noqa: E402
```

Comment:
Will `os.path.realpath(__file__)` always work on all platforms? Maybe using the xbmc Python api for this is safer:
```python
from kodi_six import xbmc, xbmcaddon
sys.path.append(os.path.join(xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path')), 'lib'))
```

Reply:
Yes, using os.path.realpath(__file__) is the best way to do it.

Reply:
Okay, but I hope we don't  break the rules using `os.path.realpath(__file__)`
https://kodi.wiki/view/Add-on_rules#Requirements_for_scripts_and_plugins

Reply:
I see other add-ons doing this too, but you can never be sure.